src/components/IndividualPageContent/LearnMoreTooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import HelpOutlineOutlinedIcon from "@mui/icons-material/HelpOutlineOutlined";
+import {Box, Link, Typography, Tooltip, useTheme} from "@mui/material";
+import {grey} from "../../themes/colors/aptosColorPalette";
+import {Stack} from "@mui/system";
+
+function TooltipBox({children}: {children?: React.ReactNode}) {
+  return <Box sx={{width: 25}}>{children}</Box>;
+}
+
+type LearnMoreTooltipProps = {
+  text: string;
+  link?: string;
+  linkToText?: boolean;
+};
+
+export function LearnMoreTooltip({
+  text,
+  link,
+  linkToText,
+}: LearnMoreTooltipProps): JSX.Element {
+  // TODO: unify colors for the new transaction page
+  const theme = useTheme();
+  const color = theme.palette.mode === "dark" ? grey[600] : grey[200];
+
+  return (
+    <TooltipBox>
+      <Tooltip
+        title={
+          <Stack alignItems="flex-end">
+            {linkToText ? (
+              <Link alignSelf="flex-end" href={link} color="inherit">
+                {text}
+              </Link>
+            ) : (
+              <>
+                <Typography variant="inherit">{text}</Typography>
```

Comment:
Monospace doesn't seem like the write font choice for a tooltip.

Reply:
Agreed. If we could specify Tooltips to utilize **apparat** that would be ideal. Understandably the default font config for Explorer is **lft-etica-mono**, so we should probably create a `Typography` variant with `font-family:"apparat";`

Reply:
Working on it.